chart/templates/logging-backends.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apm-hub.flanksource.com/v1
+kind: LoggingBackend
+metadata:
+  name: kubernetes
+spec:
+  backends:
+  - kubernetes:
+      kubeconfig: {}
+      routes:
+        - idPrefix: ''
```

Comment:
Need to add - type: KubernetesNode and another backed type: KubernetesPod

Reply:
@moshloop 👍🏽